src/Features/Core/Portable/Diagnostics/Analyzers/NamingStyleDiagnosticAnalyzerBase.cs

```diff
@@ -149,8 +149,7 @@ void SyntaxNodeAction(SyntaxNodeAnalysisContext syntaxContext)
             return DiagnosticHelper.Create(Descriptor, symbol.Locations.First(), applicableRule.EnforcementLevel, additionalLocations: null, builder.ToImmutable(), failureReason);
         }
 
-        [PerformanceSensitive("https://github.com/dotnet/roslyn/issues/23582", OftenCompletesSynchronously = true)]
-        private static async ValueTask<NamingStylePreferences> GetNamingStylePreferencesAsync(
+        private static NamingStylePreferences GetNamingStylePreferencesAsync(
```

Comment:
Nit: rename to `GetNamingStylePreferences`. Was the removal of `PerformanceSensitiveAttribute` intentional? Tagging @sharwell

Reply:
It was not. I'll revert.